### DISCUSSION TITLE
Hot reloading with Colima

### DISCUSSION THREAD

**Discussion by vitorf7** [abiosoft/colima#637] [Q&A] [answered]
Hi everyone I was wondering if there is any way to do hot reloading on a mac with colima.
In previous project at another company we had our Golang projects use [reflex](https://github.com/cespare/reflex) but we also used Docker Desktop. 
Recently while at a new company where we only use colima, I have tried to do the same but it does not seem to be working as the project is not reloaded.

Any help would be much appreciated. Thanks

**Comment by abiosoft** [comment]
As a temporary workaround you should be able to configure reflex to run `colima ssh -- touch`

I think the following should work fine for Go projects.
```
reflex -r '\.go$' colima ssh -- xargs touch
```

**Reply by vitorf7** [reply]
Ah thank you, I'll try that and let you know.
That command would basically run reflex from the hosts machine and use the `colima ssh -- touch` command to detect changes.

Is there any workaround at this moment for a project that runs the reflex command but from inside the docker container?
In previous projects we had reflex installed on the container and then run the containers detached and with the reflex command running inside. Not sure if it's something that is possible with a workaround or maybe on the pipeline of features in the future hence the question.

Sorry for any inconvenience, and that you so much for the hard work put into Colima.

**Reply by abiosoft** [reply, marked as answer]
You can track this issue https://github.com/abiosoft/colima/issues/261.
